website/events/models/event.py

```diff
@@ -195,6 +195,10 @@ def registration_started(self):
     def registration_required(self):
         return bool(self.registration_start) or bool(self.registration_end)
 
+    @property
+    def payment_required(self):
+        return not (self.price == 0)
```

Comment:
????

```suggestion
        return self.price != 0

Reply:
![Screenshot 2020-11-24 at 20 45 34](https://user-images.githubusercontent.com/1799914/100144095-908c0680-2e96-11eb-91a4-0c98b0489443.png)

Reply:
cannot reproduce this.....